sockeye/inference.py

```diff
@@ -805,8 +848,16 @@ def _decode_step(self,
                 sequences = mx.nd.slice_axis(sequences, axis=1, begin=0, end=target_max_length)
 
         model_probs, model_attention_probs, model_states = [], [], []
-        for model, state in zip(self.models, states):
-            probs, attention_probs, state = model.run_decoder(sequences, bucket_key, state)
+        for model, out_w, out_b, state in itertools.zip_longest(
```

Comment:
good catch with the izip longest. I assume this is because not all models may have caching enabled and could return None here. 
Could you add a comment for this?

Reply:
Added.